CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,44 @@
+
+
+## Our Standards
+Examples of behavior that contributes to creating a positive environment include:
+
+### Team participation:
+- Improving my team's velocity is more important than improving my individual velocity.
+-	I will ask for help when I need it.
+-	I will provide help when asked.
+-	I will be honest at all times.
+-	If I don't believe something is possible I will say so without hesitation.
+-	I will be respectful of differing viewpoints and experiences
+
+### Peer reviews
+-	I will be professional and tactful when providing review comments.
+-	I will be open to receiving feedback when others review my work.
+
+### Writing code
+-	I will express myself through code.
+-	I will add comments to code only if I have failed to express myself through code.
+-	I will write code that adheres to my team's formatting and style conventions.
+-	I will not submit production code without thorough in-memory unit tests.
+-	I will not submit code that causes any tool to emit a warning.
+
+# Examples of unacceptable behaviour:
+-	Trolling, insulting/derogatory comments, and personal or political attacks
+-	Public or private harassment
+-	Publishing others' private information, such as a physical or electronic address, without explicit permission
+-	Other conduct which could reasonably be considered inappropriate in a professional setting
+
+
+## Our Responsibilities
+Project maintainers are responsible for clarifying the standards of acceptable behavior and are expected to take appropriate and fair corrective action in response to any instances of unacceptable behavior.
+Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
+## Scope
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
+## Enforcement
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at {{ email }}. All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```

Comment:
Maybe add an actual email here, or a github username

Reply:
Good point. I wanted to ask if we should ask Nikolai to do this.